models/migrations.js

```diff
@@ -0,0 +1,40 @@
+const firestore = require("../utils/firestore");
+const userModel = firestore.collection("users");
+const cardColorIdArray = require("../constants/cardColorIdArray");
+const { getRandomIndex } = require("../utils/helpers");
+/**
+ * Adds default colors property
+ * @return {Promise<usersMigrated|Object>}
+ */
+
+const addDefaultColors = async () => {
+  try {
+    const usersSnapshot = await userModel.get();
+    const migratedUsers = [];
+    const updateUserPromises = [];
+    const usersArr = [];
+
+    usersSnapshot.forEach((doc) => usersArr.push({ id: doc.id, ...doc.data() }));
+
+    for (const user of usersArr) {
+      const colors = user.colors ? user.colors : {};
+      if (user.colors === undefined) {
+        const userColorIndex = getRandomIndex(cardColorIdArray);
+
+        colors.color_id = cardColorIdArray[parseInt(userColorIndex)].COLOR_ID;
```

Comment:
Would it be better to handle parseInt or whether index is number in function itself?

Reply:
actually I am returning a number , but there is an es-lint issue which was popping up